pynicotine/gtkgui/search.py

```diff
@@ -887,6 +902,9 @@ def check_filter(self, row):
         if filters["filtertype"] and not self.check_file_type(filters["filtertype"], row[11]):
             return False
 
+        if filters["filterlength"] and not self.check_digit(filters["filterlength"], row[16].get_uint64(), False):
```

Comment:
```suggestion
        if filters["filterlength"] and not self.check_digit(filters["filterlength"], row[16].get_uint(), False):
```

Reply:
Agreed it is adequate to allow for files which are up to 68 years long!